hooks/pre/30-upgrade.rb

```diff
@@ -1,17 +1,23 @@
+def start_service(service)
+  status = Kafo::Helpers.execute("service-wait #{service} status")
```

Comment:
We could, I'll have to check. Mostly worried about what upstart does when you try to start an already started service, I think that's why I check the status first here.  systemd will return 0, but all of the old init scripts might not play so nicely.